src/Sharpbrake.Client/Utils.cs

```diff
@@ -204,6 +206,54 @@ public static IList<Regex> CompileRegex(IList<string> stringPatterns)
             return stringPatterns?.Select(pattern => new Regex(pattern, RegexOptions.Compiled | RegexOptions.IgnoreCase)).ToList();
         }
 
+#if NETSTANDARD1_4
+        private static readonly object logResponseLocker = new object();
+#endif
+
+        /// <summary>
+        /// Logs response from Airbrake to the file.
+        /// </summary>
+        public static void LogResponse(string logFile, AirbrakeResponse response)
+        {
+            if (response == null)
+                return;
+
+#if NETSTANDARD1_4
+            using (var writer = File.AppendText(logFile))
+            {
+                lock (logResponseLocker)
+                {
+                    writer.WriteLineAsync(
+                        $"[{DateTime.UtcNow.ToString("o", CultureInfo.InvariantCulture)}] {response.Status} {response.Id} {response.Url}");
+                }
+            }
+#else
+            using (var writer = TextWriter.Synchronized(File.AppendText(logFile)))
+                writer.WriteLineAsync(
+                    $"[{DateTime.UtcNow.ToString("o", CultureInfo.InvariantCulture)}] {response.Status} {response.Id} {response.Url}");
+#endif
+        }
+
+        /// <summary>
+        /// Gets message with properties positionally formatted into the message template
+        /// using culture-specific formatting information.
+        /// </summary>
+        /// <param name="provider">Object that supplies culture-specific formatting information.</param>
+        /// <param name="messageTemplate">Message template that represents .NET composite format string.</param>
+        /// <param name="propertyValues">Zero or more properties to format.</param>
+        /// <returns></returns>
+        public static string GetMessage(IFormatProvider provider, string messageTemplate, params object[] propertyValues)
+        {
+            // TODO: Add support for message templates for structured logging (https://messagetemplates.org/)
```

Comment:
I guess this will be addressed in a separate PR? Just pinging you in case you forgot to resolve this TODO (maybe not 😁).

Reply:
It's a long story... I spent 2 days just on tokenizer (attach "meaning" to words from the template). And it still failed on edge cases (like [this](https://github.com/serilog/serilog/blob/dev/test/Serilog.Tests/Parsing/MessageTemplateParserTests.cs#L89) or [this](https://github.com/serilog/serilog/blob/dev/test/Serilog.Tests/Parsing/MessageTemplateParserTests.cs#L158)). The best implementation in the field that I found is in Serilog but it's quite complicated and the code just for parsing the message template with properties evaluation is several times bigger than all our codebase including integrations. So I decided to freeze that part for now. Maybe we will return to it in the future. As for now, it requires too many efforts for the value it brings (in our case). I decided to go with the standard .NET format strings (+ string interpolations are supported as well) that with the Airbrake backend do a quite good job.